### ISSUE TITLE
Console: Nonlinear system: shows same approximation order/type but different curly braces

### ISSUE THREAD

**Issue by mangerij** [idaholab/moose#5305]
I've noticed this for a while, never really paid much attention to this but Cody told me to put an issue up just in case.

Nonlinear System:
  Num DOFs:                25363
  Num Local DOFs:          25363
  Variables:               { "polar_x" "polar_y" "polar_z" } { "potential_int" "potential_ext" "disp_x" 
                             "disp_y" "disp_z" } 
  Finite Element Types:    "LAGRANGE" "LAGRANGE" 
  Approximation Orders:    "FIRST" "FIRST" 

In other words, why is polar_x,y,z not listed in the same curly brace with the others?

Screenshot below
![screenshot from 2015-06-24 15 19 32](https://cloud.githubusercontent.com/assets/7597716/8341258/4879e0d2-1a89-11e5-8de0-9692a5fe3b09.png)

**Comment by permcody** [comment]
Unrelated but since I'm staring at your screen shot, you should probably get rid of those legacy flags in your application. You can remove them in your `FerretApp.C` file. You might see a few diffs if you have Postprocessors turned on, but the new behavior makes more sense than the old behavior. Also you can restore the old behavior for both of these flags with explicit use of additional `execute_on` flags if you wish.

**Comment by YaqiWang** [comment]
Are those variables restricted on different blocks?

**Comment by mangerij** [comment]
Yeah. I suppose that answers the question! 

So does that mean when people have hundreds of blocks, they have hundreds of groups of variables in this list? That seems a bit cumbersome.

**Comment by dschwen** [comment]
The output is per variable group and not per order/type.

**Comment by permcody** [comment]
Well yes, but his question is asking what makes a variable group. I believe
we have the answer.

**Comment by permcody** [comment]
@mangerij are you satisfied with that answer? If so we'll close this ticket unless you feel like we should make an adjustment somewhere.

**Comment by mangerij** [comment]
Yes that is fine.
